### PR TITLE
Add net-snmp IP-MIB

### PR DESCRIPTION
**Pull request by ibigbug**
To walk https://oidref.com/1.3.6.1.2.1.4.20

This can be useful when walking IP address of interfaces

### CODE REVIEW

**Comment by ibigbug** [comment]
ping

**Comment by SuperQ** [comment]
We typically only add new MIBs when we need them for the generator.yml. Would you mind adding a config to it?

For example, the OID you mentioned is deprecated, but `ipv4InterfaceTable` might be one to add.

**Comment by ibigbug** [comment]
added `ipv4InterfaceTable` to `IF_MIB`

File: generator/generator.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 modules:
   # Default IF-MIB interfaces table with ifIndex.
   if_mib:
-    walk: [sysUpTime, interfaces, ifXTable]
+    walk: [sysUpTime, interfaces, ifXTable, ipv4InterfaceTable]
```
Comment (SuperQ) [inline generator/generator.yml:4]: Please add this as a separate module named `ip_mib`.

**Review by SuperQ — CHANGES_REQUESTED** [review]

**Comment by SuperQ** [comment]
You will need to run the generator to update the `snmp.yml` example in the repo.

**Comment by ibigbug** [comment]
not sure `fixed_size: 6` is getting removed, same thing in https://github.com/prometheus/snmp_exporter/pull/681/files

File: snmp.yml [hunk 2]
```diff
@@ -4636,7 +4636,6 @@ cisco_wlc:
     indexes:
     - labelname: bsnAPDot3MacAddress
       type: PhysAddress48
-      fixed_size: 6
```
Comment (SuperQ) [inline snmp.yml:4639]: It looks like your generator or mibs are out of date. These should not be removed.

Comment (ibigbug) [reply snmp.yml:4639]: not sure - I've rebased the latest main branch of this repo and the mibs were just generated with make mibs.

any ideas?

**Comment by SuperQ** [comment]
This still needs to be fixed up.

**Comment by ibigbug** [comment]
ty @SuperQ the generator is working now

**Review by SuperQ — APPROVED** [review]

**Pull request merged**